packages/ai-chat-ui/src/browser/chat-input-widget.tsx

```diff
@@ -460,10 +460,10 @@ const ChangeSetBox: React.FunctionComponent<{ changeSet: ChangeSetUI }> = ({ cha
                 <button
                     className='theia-button'
                     disabled={changeSet.disabled}
-                    title={nls.localize('theia/ai/chat-ui/acceptAll', 'Accept all pending changes')}
-                    onClick={() => changeSet.acceptAllPendingElements()}
+                    title={nls.localize('theia/ai/chat-ui/acceptAll', 'Apply all pending suggestions')}
+                    onClick={() => changeSet.applyAllPendingElements()}
                 >
-                    Accept
+                    {nls.localizeByDefault('Copy All')}
```

Comment:
```suggestion
                    {nls.localizeByDefault('Apply All')}
```

Reply:
Seems to have been changed by a `lint --fix` invocation - switched to the more verbose non-default form.